compiler/luci-micro/import_libs.py

```diff
@@ -0,0 +1,32 @@
+import shutil, re
+
+shutil.copytree('../stdex', './lib/stdex', dirs_exist_ok=True)
+shutil.copytree('../oops', './lib/oops', dirs_exist_ok=True)
+shutil.copytree('../pepper-str', './lib/pepper-str', dirs_exist_ok=True)
+shutil.copytree('../pepper-str', './lib/pepper-str', dirs_exist_ok=True)
+shutil.copytree('../logo', './lib/logo', dirs_exist_ok=True)
+shutil.copytree('../logo-core', './lib/logo-core', dirs_exist_ok=True)
+shutil.copytree('../loco', './lib/loco', dirs_exist_ok=True)
+shutil.copytree('../locomotiv', './lib/locomotiv', dirs_exist_ok=True)
+shutil.copytree('../angkor', './lib/angkor', dirs_exist_ok=True)
+shutil.copytree('../luci/lang', './lib/luci/lang', dirs_exist_ok=True)
+shutil.copytree('../luci/import', './lib/luci/import', dirs_exist_ok=True)
+shutil.copytree('../luci/env', './lib/luci/env', dirs_exist_ok=True)
+# shutil.copytree('../luci/log', './lib/luci/log', dirs_exist_ok=True)
```

Comment:
Do I understand it right, that you copy luci-interpreter (and all dependencies) in this directory, and do not modify original code?